core/management/commands/purge_essays.py

```diff
@@ -1,15 +0,0 @@
-from django.core.management.base import BaseCommand
```

Comment:
I think we may want to keep this, as several organizations are still using essays.  Just because we can't load them currently doesn't mean that we don't want to flush them!